oughta.cabal

```diff
@@ -26,7 +26,7 @@ source-repository head
 
 common shared
   default-language: GHC2021
-  build-depends: base >= 4.17 && < 5
+  build-depends: base >= 4.19 && < 5
```

Comment:
I'm a bit worried about this, since this will prevent the library from building with anything older than GHC 9.8, which is likely within our support window for certain things.

Reply:
I was reluctant to claim that it would build with older GHCs given that they are not tested in CI: https://github.com/GaloisInc/oughta/blob/71d371e8f7db2c91dcab30707f76f79df9a3009d/.github/workflows/test.yml#L11

But you're right, this will prevent Oughta from working even with GREASE. We should test with more versions of GHC and add a release that is compatible with them: https://github.com/GaloisInc/oughta/issues/9